go/ql/lib/semmle/go/frameworks/JWT.qll

```diff
@@ -0,0 +1,91 @@
+/** Provides models of commonly used functions in common JWT packages. */
+
+import go
+
+/**
+ * Provides models of commonly used functions in common JWT packages.
+ */
+module JWT {
+  /** Gets the package name `github.com/lestrrat-go/jwx` v2. */
+  string packageLestrrat() { result = package("github.com/lestrrat-go/jwx/v2/jwt", "") }
```

Comment:
```suggestion
  string packageLestrrat() { result = "github.com/lestrrat-go/jwx/v2/jwt"
```
No need to use the `package` predicate, since you are specifying an exact version.